xbmc/FileItem.cpp

```diff
@@ -1691,6 +1691,15 @@ void CFileItem::SetFromMusicInfoTag(const MUSIC_INFO::CMusicInfoTag &music)
 
   *GetMusicInfoTag() = music;
   FillInDefaultIcon();
+  if (!music.GetStationName().empty())
```

Comment:
As you are using '!music.GetStationName().empty()' as synonym for "is shoutcast" at several places, wouldn't it make sense to have a method IsShoucastStream() at MusicInfoTag?